README.md

```diff
@@ -237,6 +238,32 @@ systemctl --user enable himalaya.service
 systemctl --user start  himalaya.service
 ```
 
+## Completions
+
+```sh
+# For bash shells
+himalaya bash-completions
+
+# For zsh shells
+himalaya zsh-completions
+
+# For fish shells
+himalaya fish-completions
```

Comment:
Perhaps these should be `himalaya completions {bash, fish, zsh}` to be consistent with the rest of the subcommand structure (`himalaya template ...` for instance).

Reply:
Agree.

Reply:
I also agree, it's a good point. I made the change in the last commit (https://github.com/soywod/himalaya/commit/aad74add5296ceadda2b1be3d4ad778a51f6ecd4). I also redefine shells as arguments instead of subcommands for the completion + help.